Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -179,6 +179,35 @@ await useCase.Execute(new OrganisationQuery(name, identifier))
         return app;
     }
 
+    public static RouteGroupBuilder UseSupportEndpoints(this RouteGroupBuilder app)
+    {
+        app.MapPatch("/organisation/{organisationId}",
+                [OrganisationAuthorize([AuthenticationChannel.OneLogin])]
```

Comment:
Missing Authorization test

Reply:
@dharmverma I wanted to ask you about this section actually. I assume there will also be a different role here. What will it be? Still a bit confused. The migration to add Scopes to the person table has been merged into main.

Reply:
@rmohammed-goaco I am talking about test here.
We have a ticket on board to cover the scope.
All you need to write a test case that only OneLogin users are able to access it.

Reply:
Test case added: https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/pull/652/files#diff-dfa6734ca97b6e1fc83fb71473e1995bd0e27226adf33805fd639a4a321e481e